src/test/java/org/sonatype/nexus/ci/iq/IqPolicyEvaluatorSlaveIntegrationTest.groovy

```diff
@@ -95,6 +99,40 @@ class IqPolicyEvaluatorSlaveIntegrationTest
       jenkins.assertBuildStatusSuccess(build)
   }
 
+  def 'Should perform a freestyle build on slave with new server version'() {
+    given: 'a jenkins project'
+      FreeStyleProject project = jenkins.createFreeStyleProject()
+      project.assignedNode = jenkins.createSlave()
+      project.buildersList.add(new IqPolicyEvaluatorBuildStep('stage', new SelectedApplication('app'), [], [], false, 'cred-id'))
+      configureJenkins()
+
+    and: 'a mock IQ server stub'
+      configureIqServerMock(incrementVersion(IqPolicyEvaluatorUtil.MINIMAL_SERVER_VERSION_REQUIRED))
+
+    when: 'the build is scheduled'
+      def build = project.scheduleBuild2(0).get()
+
+    then: 'the return code is successful'
+      jenkins.assertBuildStatusSuccess(build)
+  }
+
+  def 'Should not perform a freestyle build on slave with older server version'() {
+    given: 'a jenkins project'
+      FreeStyleProject project = jenkins.createFreeStyleProject()
+      project.assignedNode = jenkins.createSlave()
+      project.buildersList.add(new IqPolicyEvaluatorBuildStep('stage', new SelectedApplication('app'), [], [], false, 'cred-id'))
+      configureJenkins()
+
+    and: 'a mock IQ server stub'
+      configureIqServerMock(decrementVersion(IqPolicyEvaluatorUtil.MINIMAL_SERVER_VERSION_REQUIRED))
+
+    when: 'the build is scheduled'
+      def build = project.scheduleBuild2(0).get()
+
+    then: 'the return code is successful'
```

Comment:
Maybe `then: 'the build fails'` ?  Or are you just interested in successfully getting a return code?

Reply:
>Maybe then: 'the build fails' ? Or are you just interested in successfully getting a return code?

Actually, I think we want to verify failure here.  Updated the `then` message for failures in 48311ed855396c55f3465376be249d7cdd7f7947